geemusic/__init__.py

```diff
@@ -1,14 +1,17 @@
+from __future__ import absolute_import
```

Comment:
is this a reason for switching to absolute imports? I haven't kept up with python best practices as much lately (esp with py3 gaining traction).